README.md

```diff
@@ -155,10 +155,10 @@ html
 
 ## Contributors
 
-- [eliasjpr](https://github.com/eliasjpr) Elias Perez - creator, maintainer
+- [eliasjpr](https://github.com/eliasjpr) Elias Perez - Maintainer
 - [fridgerator](https://github.com/fridgerator) Nick Franken - Maintainer
 - [elorest](https://github.com/elorest) Isaac Sloan - Maintainer
-- [phoffer](https://github.com/phoffer) Paul Hoffer - Maintainer
+- [drujensen](https://github.com/drujensen) Dru Jensen - Maintainer
 - [bew](https://github.com/fridgerator) Benoit de Chezelles - Member
```

Comment:
Ahh! Sorry @bew I read this from my phone I will correct your github tag my apologies my friend